app/views/govuk_publishing_components/components/docs/layout_header.yml

```diff
@@ -60,6 +60,27 @@ examples:
       - text: News and communications
         href: "item-6"
         active: true
+  with_custom_navigation_aria_label:
+    description: The navigation has `aria-label="Top level"` by default. This option is here for when the `aria-label` needs to be more descriptive than that.
+    data:
+      search_left: true
+      navigation_items_label: "Departments and policy"
```

Comment:
At the risk of making this option name too verbose, should it include the word `aria`?

Reply:
I think we can include the word `aria` without making it too verbose. I'm thinking it can be changed to `navigation_aria_label` since the word "items" there doesn't really provide much value, IMO.